incubator/kubeless/templates/kafka-svc.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: kafka
+  namespace: kubeless
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+spec:
+  ports:
+  - name: kafka-port
+    port: 9092
+    targetPort: kafka-port
+  selector:
```

Comment:
Add labels `app` and `release` to selector.

Reply:
done